tests/flow/test_constraint.py

```
@@ -0,0 +1,1061 @@
+        # the WAIT command forces master slave sync to complete
+        self.source.execute_command("WAIT", 1, 0)
+
+    def monitor_thread(self):
+        global MONITOR_ATTACHED
+        try:
+            with self.replica.monitor() as m:
+                MONITOR_ATTACHED = True
+                for cmd in m.listen():
+                    if 'GRAPH.CONSTRAINT' in cmd['command']:
+                        self.monitor.append(cmd)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)